testpar/t_prestart.c

```diff
@@ -115,8 +122,6 @@ main(int argc, char **argv)
     if (data_array)
         free(data_array);
 
-    nerrors += GetTestNumErrs();
```

Comment:
This test never used testframe.c functionality, so `GetTestNumErrs()` isn't useful here